test/integration/patients/worklist/schedule.js

```diff
@@ -614,6 +618,34 @@ context('schedule page', function() {
       .get('@worklists')
       .find('.app-nav__link')
       .should('have.length', 1);
+
+    cy
+      .get('[data-select-all-region]')
+      .find('.button--checkbox')
+      .should('not.exist');
+
+    cy
+      .get('[data-filters-region]')
+      .find('.schedule__filters')
+      .should('not.exist');
+
+    cy
+      .get('[data-date-filter-region]')
+      .find('div')
+      .should('not.exist');
+
+    cy
+      .get('.schedule-list__table')
+      .find('.schedule-list__list-row')
+      .its('length')
+      .should('be.gt', 0);
+
+    cy
+      .get('.schedule-list__table')
+      .find('.schedule-list__list-row .schedule-list__day-list')
+      .first()
+      .find('.schedule-list__day-list-row .js-select')
+      .should('not.exist');
   });
```

Comment:
Overview of the tests:

- Verify the page is routed to “/schedule” by default.
- Verify sidebar only contains the “Schedule” element and that is has the `is-selected` class name.
- Verify the filtering and bulk edit elements are not shown.
  - Bulk edit button, owner filter, and date filter should all not exist.
- Verify the list of actions are loaded into the page.
  - Not checking anything specific here. Just that the list contains more than `0` actions.
- Verify that individual list items do not contain the checkbox element.